configs/deblurganv2/README.md

```diff
@@ -0,0 +1,45 @@
+# DeblurGAN-v2 (ICCV'2019)
+
+> [DeblurGAN-v2: Deblurring (Orders-of-Magnitude) Faster and Better](https://arxiv.org/abs/1908.03826)
+
+> **Task**: Image Super-Resolution
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+We present a new end-to-end generative adversarial network (GAN) for single image motion deblurring, named DeblurGAN-v2, which considerably boosts state-of-the-art deblurring efficiency, quality, and flexibility. DeblurGAN-v2 is based on a relativistic conditional GAN with a double-scale discriminator. For the first time, we introduce the Feature Pyramid Network into deblurring, as a core building block in the generator of DeblurGAN-v2. It can flexibly work with a wide range of backbones, to navigate the balance between performance and efficiency. The plug-in of sophisticated backbones (e.g., Inception-ResNet-v2) can lead to solid state-of-the-art deblurring. Meanwhile, with light-weight backbones (e.g., MobileNet and its variants), DeblurGAN-v2 reaches 10-100 times faster than the nearest competitors, while maintaining close to state-of-the-art results, implying the option of real-time video deblurring. We demonstrate that DeblurGAN-v2 obtains very competitive performance on several popular benchmarks, in terms of deblurring quality (both objective and subjective), as well as efficiency. Besides, we show the architecture to be effective for general image restoration tasks too.
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://raw.githubusercontent.com/VITA-Group/DeblurGANv2/master/doc_images/pipeline.jpg"/>
+</div>
+
+## Results and models
+
+<div align="center">
+  <b> DEBLURGANv2 256x256</b>
+  <br/>
+  <img src="https://raw.githubusercontent.com/VITA-Group/DeblurGANv2/master/doc_images/kohler_visual.png" width="800"/>
+  <img src="https://raw.githubusercontent.com/VITA-Group/DeblurGANv2/master/doc_images/restore_visual.png" width="800"/>
+ </div>
+
+|                      Model                      |      Dataset       |      G Model       |  D Model   | PSNR/<br/>SSIM |                                           Download                                            |
+| :---------------------------------------------: | :----------------: | :----------------: | :--------: | :------------: | :-------------------------------------------------------------------------------------------: |
+| [fpn_inception](./deblurganv2_fpn_inception.py) | GoPro Test Dataset | InceptionResNet-v2 | double_gan |  29.55/ 0.934  | [model](https://drive.google.com/uc?export=view&id=1UXcsRVW-6KF23_TNzxw-xC0SzaMfXOaR) \\ [log](<>) |
+| [fpn_mobilenet](./deblurganv2_fpn_mobilenet.py) | GoPro Test Dataset |     MobileNet      | double_gan |  28.17/ 0.925  | [model](https://drive.google.com/uc?export=view&id=1JhnT4BBeKBBSLqTo6UsJ13HeBXevarrU) \\ [log](<>) |
+
```

Comment:
Providing brief `Quick Start` instructions here would be better. 
You can refer to https://github.com/open-mmlab/mmagic/blob/main/configs/aot_gan/README.md#quick-start

Reply:
There is a question, can I provide the link to the weight file on Baidu network disk?

Reply:
hi @xiaomile, we suggest uploading models to https://openxlab.org.cn/models and provide this link.

Reply:
OK, thank you!